src/test/java/com/farao_community/farao/dichotomy/BusinessTestCaseRaoResult.java

```diff
@@ -0,0 +1,155 @@
+package com.farao_community.farao.dichotomy;
+
+import com.farao_community.farao.commons.Unit;
+import com.farao_community.farao.data.crac_api.State;
+import com.farao_community.farao.data.crac_api.cnec.FlowCnec;
+import com.farao_community.farao.data.crac_api.network_action.NetworkAction;
+import com.farao_community.farao.data.crac_api.range_action.PstRangeAction;
+import com.farao_community.farao.data.crac_api.range_action.RangeAction;
+import com.farao_community.farao.data.rao_result_api.ComputationStatus;
+import com.farao_community.farao.data.rao_result_api.OptimizationState;
+import com.farao_community.farao.data.rao_result_api.RaoResult;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public class BusinessTestCaseRaoResult implements RaoResult {
```

Comment:
General remark : throw exceptions on unused methods instead of default implementation ?

Reply:
done